### PR TITLE
IOTEDGE-1068 Integrate platform with HiveMQ

### PR DESCRIPTION
**Pull request by JacoJooste**

### CODE REVIEW

File: examples/proof-of-concept/hivemq/run.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+echo "====================================================="
+rm -rf "$POC_DIR"/mqtt-broker/tmp && mkdir "$POC_DIR"/mqtt-broker/tmp
+rm -rf "$POC_DIR"/mqtt-client/tmp && mkdir "$POC_DIR"/mqtt-client/tmp
+cp "$DEPLOYMENT_DIR"/tmp/_wildcard.iam.example.com* "$POC_DIR"/mqtt-broker/tmp/
+cp "$DEPLOYMENT_DIR"/tmp/_wildcard.iam.example.com* "$POC_DIR"/mqtt-client/tmp/
+cd "$POC_DIR"
+AM_IP_ADDRESS=$(minikube ip) docker-compose up -d --build
+
+# Attach to the broker
+docker attach hivemq
```
Comment (LimaEchoCharlie) [inline examples/proof-of-concept/hivemq/run.sh:45]: Is this so that we can read the HiveMQ logs and see that a Thing is publishing data?

**Review by LimaEchoCharlie — APPROVED** [review]
🥇

File: examples/proof-of-concept/hivemq/run.sh (hunk 1 above)
Comment (JacoJooste) [reply examples/proof-of-concept/hivemq/run.sh:45]: It shows the download of the JWK set and the client connecting, but not the data publish.

Comment (LimaEchoCharlie) [reply examples/proof-of-concept/hivemq/run.sh:45]: Understood. Is there a admin\demo use that we could use to connect to the broker locally?

Comment (JacoJooste) [reply examples/proof-of-concept/hivemq/run.sh:45]: The HiveMQ Control Center could be useful, but I couldn't get it working today. Will have another go if we do a more in depth demo at some point.
https://www.hivemq.com/docs/hivemq/4.4/control-center/introduction.html

**Pull request merged**